### PR TITLE
WIP: Run code through style guide checks

### PR DESCRIPTION
**Pull request by uLan08**
# Addresses #30 

Hi, @thinkingserious. This is still WIP. I just want my initial changes be reviewed and see if I am on the right track.

### Checklist
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ ] I have read the [Contribution Guide] and my PR follows them.
- [ ] I updated my branch with the master branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added in line documentation to the code I modified

### Short description of what this PR does:
-

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/dx-automator?pullRequest=33) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/dx-automator?pullRequest=33) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/dx-automator?pullRequest=33) it.</sub>

**Comment by thinkingserious** [comment]
Hello @uLan08,

Could you please [sign our CLA](https://cla.sendgrid.com/sendgrid/dx-automator?pullRequest=33) so that we may merge your work? Thanks!

With Best Regards,

Elmer

**Comment by uLan08** [comment]
Hey @thinkingserious , I just signed it but this PR is still WIP though.

File: services/tasks/project/api/models.py [hunk 1]
```diff
@@ -28,9 +29,9 @@ class Task(db.Model):
     date_multiplier = db.Column(db.Integer, nullable=False)
 
     def __init__(self, creator, link, title="", due_date=None, task_type="", category="",
-    maintainer=None, language=None, customers_count=1, estimated_custmomer_points=None,
-    estimated_points=None, impact=None, reach=1, effort=None, confidence="",
-    date_multiplier=1):
+                 maintainer=None, language=None, customers_count=1, estimated_custmomer_points=None,
```
Comment (42B) [inline services/tasks/project/api/models.py]:
```python
estimated_custmomer_points
              ^
```

**Comment by thinkingserious** [comment]
Hello @uLan08,

Could you please update `pep8` to `pycodestyle` per @42B's review? Thanks!

With Best Regards,

Elmer

**Comment by thinkingserious** [comment]
Hello @uLan08,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by uLan08** [comment]
Hi @thinkingserious ,

Oh cool! I just filled up the form, looking forward to it. Thanks!

**Pull request merged**